chapters/en/slides/module2/module2_02.Rmd

```diff
@@ -0,0 +1,279 @@
+and when they are a suitable alternative to use
+instead of line charts.
+
+---
+
+## Global Development Data
+
+| Column                | Description                                                                                  |
+|-----------------------|----------------------------------------------------------------------------------------------|
+| country               | Country name                                                                                 |
+| year                  | Year of observation                                                                          |
+| population            | Population in the country at each year                                                       |
+| region                | Continent the country belongs to                                                             |
+| sub_region            | Sub-region the country belongs to                                                            |
+| income_group          | Income group [as specified by the world bank in 2018]                                        |
+| life_expectancy       | The mean number of years a newborn would <br>live if mortality patterns remained constant    |
+| income                | GDP per capita (in USD) <em>adjusted <br>for differences in purchasing power</em>            |
+| children_per_woman    | Average number of children born per woman                                                    |
+| child_mortality       | Deaths of children under 5 years <break>of age per 1000 live births                          |
+| pop_density           | Average number of people per km<sup>2</sup>                                                  |
+| co2_per_capita        | CO2 emissions from fossil fuels (tonnes per capita)                                          |
+| years_in_school_men   | Mean number of years in primary, secondary,<br>and tertiary school for 25-36 years old men   |
+| years_in_school_women | Mean number of years in primary, secondary,<br>and tertiary school for 25-36 years old women |
+
+[as specified by the world bank in 2018]: https://datahelpdesk.worldbank.org/knowledgebase/articles/378833-how-are-the-income-group-thresholds-determined
+
+Notes:
+We will be visualizing global health and population data for a number of countries.
+This is the same data we are working with in the assignments for the first two modules.
+
+We will be looking at many different data sets in later labs,
+but we're sticking to a familiar one for now
+so that we can focus on laying down a solid understanding
+of the visualization principles
+with data we already know.
+
+## Reading in the data with pandas
+
+```{python}
+import pandas as pd
+
+
+url = 'https://raw.githubusercontent.com/UofTCoders/workshops-dc-py/master/data/processed/world-data-gapminder.csv'
+gm = pd.read_csv(url, parse_dates=['year'])
+gm = gm[gm['country'].isin(gm[gm['year'] == '2018'].nlargest(10, ['population'])['country'])]
```

Comment:
Filter data because reticulate doesn't support the Altair workaround for over 5k rows in the dataset.

Reply:
Added a todo to revisit later